app/src/main/java/com/example/travellens/FeedMainActivity.java

```diff
@@ -23,13 +30,19 @@
 import com.parse.ParseUser;
 
 public class FeedMainActivity extends AppCompatActivity {
+    private ImageView ivIcon;
     private BottomNavigationView bottomNavigationView;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_feed_main);
-
+        ivIcon = findViewById(R.id.ivIconShowOnProf);
+        AutocompleteSupportFragment autocompleteFragment = (AutocompleteSupportFragment)
+                this.getSupportFragmentManager().findFragmentById(R.id.afSearchAPI);
+        autocompleteFragment.getView().setEnabled(true);
+        autocompleteFragment.getView().setVisibility(View.VISIBLE);
+        ((EditText)autocompleteFragment.getView().findViewById(com.google.android.libraries.places.R.id.places_autocomplete_search_input)).setHintTextColor(getResources().getColor(R.color.silver));
```

Comment:
You can just store autocompleteFragment.getView() in a variable since you're having to access it multiple times.